packages/card/README.md

```diff
@@ -124,168 +160,72 @@ Cards can be supplied an `actions` via a names slot.
 </sp-card>
 ```
 
-### No Preview Image
-
-Cards with no preview image can contain a heading, a subheading, and a footer.
-
-```html demo
-<sp-card heading="Card Title" subheading="No Preview Image">
-    <div slot="footer">Footer</div>
-</sp-card>
-```
+### Options
 
-### Quiet
+#### Horizontal
 
-Quiet cards can contain a heading, a subheading, a cover photo, a description, and a footer.
+Cards with a `horizontal` attribute can contain a heading, a subheading, a cover photo, and a description.
 
 ```html
-<div style="width: 208px; height: 264px">
-    <sp-card variant="quiet" heading="Card Heading" subheading="JPG Photo">
-        <img alt="" slot="preview" src="https://picsum.photos/200/300" />
-        <div slot="description">10/15/18</div>
-        <div slot="footer">Footer</div>
-    </sp-card>
-</div>
+<sp-card horizontal heading="Card Heading" subheading="JPG Photo">
+    <img alt="" slot="cover-photo" src="https://picsum.photos/200/250" />
+    <div slot="description">10/15/18</div>
+</sp-card>
 ```
 
-When leveraging the `asset` attribute, a card can be declared as representing a `file`:
+#### Variant
 
-```html
-<div style="width: 208px; height: 264px">
-    <sp-card
-        variant="quiet"
-        heading="Card Heading"
-        subheading="JPG Photo"
-        asset="file"
-    >
-        <img alt="" slot="preview" src="https://picsum.photos/200/300" />
-        <div slot="heading">File Name</div>
-        <div slot="description">10/15/18</div>
-        <div slot="footer">Footer</div>
-    </sp-card>
-</div>
-```
+There are multiple card variants to choose from in Spectrum. The `variant`
+attribute controls the main variant of the card.
 
-Or a `folder`:
+Cards with `variant="quiet"` can contain a heading, a subheading, a cover photo, a description, and a footer. Quiet cards will also accept `actions` via a named slot.
 
 ```html
-<div style="width: 208px; height: 264px">
-    <sp-card variant="quiet" subheading="JPG Photo" asset="folder">
-        <img alt="" slot="preview" src="https://picsum.photos/200/300" />
-        <div slot="heading">Folder Name</div>
-        <div slot="description">10/15/18</div>
-        <div slot="footer">Footer</div>
-    </sp-card>
-</div>
+<sp-card variant="quiet" heading="Card Heading" subheading="JPG Photo">
+    <img alt="" slot="preview" src="https://picsum.photos/200/350" />
+    <div slot="description">10/15/18</div>
+    <sp-action-menu
+        label="More Actions"
+        slot="actions"
+        placement="bottom-end"
+        quiet
```

Comment:
Why is the `quiet` a boolean on `<sp-menu>` but a `variant` attribute on `<sp-card>`? This feels like an inconsistency in our API.

Reply:
We should create an issue to write an RFC that specifies how we treat this attribute, add a TODO with the issue number